docs/fileset/README.md

```diff
@@ -187,15 +256,28 @@ $ curl -s
 
 ## Delete a File (metadata)
 
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
 <template v-slot:js>
 
 ``` js
 jfs.fileset("fileset_name")  
  .delete()  
  .where(field => field("size").isGreaterThan(1024000))  
- .subscribe();
+ .subscribe(data=>{},err=>{});
 ```
+</template>
+<template v-slot:py>
+
+``` py
+  res = client.request(
+          method='DELETE',
+          url='/fs/<fileset-name>',
+          cond=[{"field":"size"},">",1024000]
+        ) 
+  print(res)
```

Comment:
I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

Reply:
will add next release